modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,131 @@
+= Configure Role-based Access Control
+:description: Role-based access controls provide an alternative to ACLs for managing permissions.
```

Comment:
nitpick: This feels tricky to express, but RBAC is not technically an _alternative_ to ACLs. You still need the ACLs, but instead of binding them to individual users, you bind ACLs to a role and assign users to the role. Idk if that's a meaningful distinction for public docs though.

Reply:
What if I rephrase it as, "...provides an extension to ACLs for managing permissions at scale."

Reply:
LGTM